src/pages/components/link/usage.mdx

```diff
@@ -174,33 +262,112 @@ icons.
 <Row>
 <Column colLg={8}>
 
-![inline link example](images/03_link_inline_608.png)
+![Example shows inline links within the body text](images/03_link_inline_608.png)
 
 </Column>
 </Row>
 
+<Caption>Example shows inline links within the body text</Caption>
+
 ## Modifiers
 
 ### Visited style
 
 By default, the link component does not use a visited style. Visited links
 indicate that a user has already opened the link so they can be a helpful
 indicator during task completion. Visited styles should be used sparingly
-because they often clutter the the page and add further visual noise as users
-are trying to navigate a product. They can be used if it is important that a
-user knows they have already clicked on a link.
+because they often clutter the page and add further visual noise as users are
+trying to navigate a product. They can be used to show users they have already
+clicked on a link if it's important.
+
+<Row>
+<Column colLg={8}>
+
+![Example shows unvisited and visited links](images/link-usage-modifiers-1.png)
+
+</Column>
+</Row>
+
+<Caption>Example shows unvisited and visited links</Caption>
 
 ### Links that trigger actions
 
 Some links trigger actions to aid task completion in addition to navigation.
 These links should still serve a navigation purpose. A common example is linking
 phone numbers so clicking the website automatically opens and calls the phone
-number when clicked. The label and any accompanying icons should make it clear
+number when clicked. The link text and any accompanying icons should clarify
 what action will be triggered and where the user will be directed.
 
+<Row>
+<Column colLg={8}>
+
+![Example shows inline links for email](images/link-usage-modifiers-2.png)
```

Comment:
The word GitHub in this image needs a capital H. And there should not be a space between the word and the period.
<img width="570" alt="Screenshot 2024-02-09 at 2 23 11 PM" src="https://github.com/carbon-design-system/carbon-website/assets/43969356/69d5fc4d-52a5-4bce-8b14-f8b9c6f1b941">